pkg/health/server/prober.go

```diff
@@ -250,6 +251,13 @@ func (p *prober) httpProbe(node string, ip string, port int) *models.Connectivit
 			scopedLog.WithField("rtt", rtt).Debug("Greeting successful")
 			result.Status = ""
 			result.Latency = rtt.Nanoseconds()
+
+			metrics.Latency.WithLabelValues(
+				node,
+				"",
+				ip,
+				"http",
+			).Observe(rtt.Seconds())
```

Comment:
Awesome, looks like `Seconds()` should grab it in floating point which seems right, the metric can be standardized in seconds but we should have the same degree of accuracy as elsewhere.

How do you want to represent if the probe fails? latency=0? latency=math.MaxFloat64?